src/intentLogic/getHeadlines.js

```diff
@@ -0,0 +1,40 @@
+const get = require("simple-get-promise").get;
+const asJson = require("simple-get-promise").asJson;
+
+const helpers = require('../helpers');
+const speech = require('../speech').speech;
+const sound = require('../speech').sound;
+
+const config = require("../../tmp/config.json");
+const CAPI_API_KEY = config.capi_key;
+
+module.exports = function () {
+	this.event.session.attributes.lastIntent = 'GetHeadlines';
+	this.event.session.attributes.positionalContent = [];
+
+	// TODO: check the location - US, UK, AU or International?
+	var capi_query = helpers.capi_query('uk', 'show-editors-picks=true&show-fields=standfirst,byline,headline&tag=type/article,tone/news,-tone/minutebyminute');
+
+	get(capi_query)
+		.then(asJson)
+		.then((json) => {
+			if (json.response.editorsPicks && json.response.editorsPicks.length > 1) {
+				var headlines_speech = speech.acknowledgement + speech.headlines.top;
+				[0, 1, 2].forEach((i) => {
+					headlines_speech += json.response.editorsPicks[i].fields.headline
+						+ sound.break
+						+ json.response.editorsPicks[i].fields.standfirst.replace(/<(?:.|\n)*?>/gm, '').replace('\u2022', '.')
```

Comment:
You could add a comment here or write the code in such a way that makes it clearer what it's doing.